platform/fetch-native/test/integration/index.test.ts

```diff
@@ -0,0 +1,5 @@
+import 'mocha'
+
+describe.skip('PlatformFetch', () => {
+  // TODO: Write actual tests
```

Comment:
Is this PR still wip? Because it's unlikely this TODO will ever be completed if this is merged as-is. Either this will not ever have tests or it gets merged with them.

Reply:
Tests were not called anyway, removed these files.